assets/src/application/services/utilities/date/misc.ts

```diff
@@ -12,5 +12,10 @@ export const setTimeToZeroHour = (date: Date): Date => pipe(setHours(0), setMinu
 export const setDateToToday = (date: Date): Date => {
 	const today = new Date();
 
-	return pipe(setDate(today.getDate()), setMonth(today.getMonth()), setYear(today.getFullYear()))(date);
+	// prettier-ignore
```

Comment:
instead of ignore another solution:

![image](https://user-images.githubusercontent.com/1477453/81853444-14b71600-9565-11ea-9d54-fdaeb614cf8c.png)